tools/cachemgr.cc

```diff
@@ -693,7 +693,7 @@ read_reply(int s, cachemgr_request * req)
             }
 
             istate = isActions;
-        /* yes, fall through, we do not want to loose the first line */
+            /* [[fallthrough]] we do not want to loose the first line */
```

Comment:
If we are manually editing this, we might as well fix the spelling:

```suggestion
            /* [[fallthrough]] we do not want to lose the first line */
```